### PR TITLE
fix: forcePopupAlign is align to forceAlign instead

### LINKED ISSUE #41532: Slider组件第一次显示Tooltip时报错： Warning: [antd: Tooltip] `forcePopupAlign` is align to `forceAlign` instead.
### Reproduction link

[https://79gldi.csb.app/](https://79gldi.csb.app/)

### Steps to reproduce

鼠标第一次移入Slider的滑块，显示Tooltip时报错：Warning: [antd: Tooltip] `forcePopupAlign` is align to `forceAlign` instead.

### What is expected?

不报错

### What is actually happening?

报错

| Environment | Info |
| --- | --- |
| antd | 5.3.3 |
| React | 18.2.0 |
| System | windows10 |
| Browser | Chrome 110.0.5481.180 |

### PR DESCRIPTION
**Pull request by MuxinFeng**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [x] Other (about what?)

### 🔗 Related issue link

close https://github.com/ant-design/ant-design/issues/41532

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    forcePopupAlign is align to forceAlign instead       |
| 🇨🇳 Chinese |     “forcePopupAlign”改为“forceAlign”      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at ef929c7</samp>

Rename `forcePopupAlign` to `forceAlign` in `SliderTooltip` component and update test case. This improves the consistency and clarity of the code and ensures the test case passes.

### <samp>🤖 Generated by Copilot at ef929c7</samp>

*  Rename `forcePopupAlign` method to `forceAlign` in `SliderTooltip` component to match the underlying `Align` component from `rc-align` ([link](https://github.com/ant-design/ant-design/pull/41540/files?diff=unified&w=0#diff-fcd4af359a451a2a3eb4f0d0e9d052cf0f1a10d32cc2c9033988cea681ce88a0L21-R21))
*  Update test case for `SliderTooltip` component to use `forceAlign` method instead of `forcePopupAlign` method and verify that it is called when `keepAlign` function is triggered ([link](https://github.com/ant-design/ant-design/pull/41540/files?diff=unified&w=0#diff-0109698c15bdf20cfbc3ebbf6d97cca423b37639651868154216f3c19c4520c2L136-R143))

### CODE REVIEW

**Comment by MuxinFeng** [comment]
`slider` 这边不像 `tooltip` 会把 ref 暴露出去，这样修改不会引起 break change。

**Comment by acyza** [comment]
https://github.com/ant-design/ant-design/blob/master/components/tooltip/index.tsx#L223-L226
这个还有用吗？

**Comment by MuxinFeng** [comment]
> https://github.com/ant-design/ant-design/blob/master/components/tooltip/index.tsx#L223-L226 这个还有用吗？

image
这里应该会有影响。

**Comment by zombieJ** [comment]
Cov 还丢了一点

**Comment by acyza** [comment]
test工作流没触发吗？reopen一下看看？

File: components/tooltip/__tests__/tooltip.test.tsx [hunk 1]
```diff
@@ -590,4 +591,16 @@ describe('Tooltip', () => {
     );
     expect(container).toMatchSnapshot();
   });
+
+  it('use ref.current.forcePopupAlign', async () => {
```
Comment (yoyo837) [inline components/tooltip/__tests__/tooltip.test.tsx:595]: `forceAlign` 本身没有用例吗?

**Review by zombieJ — APPROVED** [review]

File: components/tooltip/__tests__/tooltip.test.tsx (hunk 1 above)
Comment (yoyo837) [reply components/tooltip/__tests__/tooltip.test.tsx:595]: #40632 加 warning 的时候, 是不是应该透出来呢?

Comment (yoyo837) [reply components/tooltip/__tests__/tooltip.test.tsx:595]: 我明白了, 是之前的也没透出来

**Pull request merged**